admin/admin_edit.php

```diff
@@ -84,6 +84,7 @@
             } else unset($_REQUEST['password']);
 
             if (!$admin_id) $admin_id=time();
+            foreach (array('fname','lname') as $k) $_REQUEST[$k]=trim($_REQUEST[$k]);
```

Comment:
I fixed 1. and 3. Re 2., I think there was some reason for only rimming fname and lname, though I do not remember it ... I would just leave as is for the moment, to not run into any trouble.